core/modules/admin_bar/css/admin_bar.css

```diff
@@ -137,7 +215,7 @@
 [dir="rtl"] #admin-bar .dropdown li li a {
   border-left: none;
 }
-#admin-bar .dropdown a:hover {
+#admin-bar .dropdown ul a:hover {
```

Comment:
This prevent "user" and "logout" links to have a hover style.
Moreover, every other link (nested, nested expandable, ...) have a `text-decoration: underline;`, so I'd like first level link to have it too. #a11y

Reply:
I did that on purpose (removed the underline) as I thought it was a visual improvement. I didn't realize it was an #a11y regression -- putting back!